recipes-bsp/u-boot/files/0009-dts-iot2050-Sync-kernel-dts-to-u-boot.patch

```diff
@@ -0,0 +1,1463 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Li Hua Qian <huaqian.li@siemens.com>
+Date: Tue, 29 Aug 2023 11:46:21 +0800
+Subject: [PATCH] dts: iot2050: Sync kernel dts to u-boot
+
```

Comment:
Would be helpful to note the source version here.

Reply:
...which probably is 6.1 + our patches. So this patch here is downstream-only.

Reply:
Split this patch to two? one for upstream and one for downstream?

Reply:
No, not worth to bother. We cannot predict source and target versions of upstream yet.